### PR TITLE
Remove empty sections

### LINKED ISSUE #6118: There are three (section) headlines without any content
What’s the point of these without content? If information is missing here it should at least describe what is missing.
The headlines are:
* Compiling the Code
* Robust Programming
* .NET Framework Security

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e0d1f4c0-e9a3-71b9-0bf1-ebefc8aeab94
* Version Independent ID: 86aa22d9-7e09-3226-aa7a-ff5d9c981d33
* Content: [How to: Group, Sort, and Filter Data in the DataGrid Control](https://docs.microsoft.com/en-us/dotnet/framework/wpf/controls/how-to-group-sort-and-filter-data-in-the-datagrid-control#robust-programming)
* Content Source: [docs/framework/wpf/controls/how-to-group-sort-and-filter-data-in-the-datagrid-control.md](https://github.com/dotnet/docs/blob/master/docs/framework/wpf/controls/how-to-group-sort-and-filter-data-in-the-datagrid-control.md)
* Product: **dotnet-framework**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #6118 - found a few more topics with the same issue

[Hide whitespace changes](https://github.com/dotnet/docs/pull/6126/files?utf8=%E2%9C%93&diff=split&w=1) for a better diff

### CODE REVIEW

File: docs/framework/network-programming/how-to-download-files-with-ftp.md [hunk 1]
```diff
@@ -1,55 +1,50 @@
...
+            Console.WriteLine("Download Complete, status {0}", response.StatusDescription);
+
+            reader.Close();
+            response.Close();
+        }
+    }
+}
+```
+
+## Compiling the code
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-download-files-with-ftp.md]: This section can also be deleted, References are to assemblies, not namespaces; importing the System,.Net namespace is handled by the using statement. No types in the System.Net assembly are used.

File: docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md [hunk 2]
```diff
@@ -1,55 +1,50 @@
...
+            Console.WriteLine("Directory List Complete, status {0}", response.StatusDescription);
+
+            reader.Close();
+            response.Close();
+        }
+    }
+}
+```
+
+## Compiling the code
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md]: Same comment as previously: this section can be deleted.

File: docs/framework/network-programming/how-to-upload-files-with-ftp.md [hunk 3]
```diff
@@ -48,16 +46,13 @@ namespace Examples.System.Net
...
-  
-## Robust Programming  
-  
-## .NET Framework Security
+        }
+    }
+}
+```
+
+## Compiling the code
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-upload-files-with-ftp.md]: This section can be removed.

File: docs/framework/network-programming/how-to-download-files-with-ftp.md [hunk 4]
```diff
@@ -1,55 +1,50 @@
...
+
+This sample shows how to download a file from an FTP server.
+
+## Example
+
+```csharp
+using System;
+using System.IO;
+using System.Net;
+using System.Text;
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-download-files-with-ftp.md]: the using System.Text; statement is not needed and can be deleted.

File: docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md [hunk 5]
```diff
@@ -1,55 +1,50 @@
...
+
+This sample shows how to list the directory contents of an FTP server.
+
+## Example
+
+```csharp
+using System;
+using System.IO;
+using System.Net;
+using System.Text;
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md]: the using System.Text; statement is not needed and can be deleted.

**Review by rpetrusha — APPROVED** [review]
I left a couple of comments, @mairaw, about removing 2 using statements and the sections on compiling the code. You can then merge when you're ready.

**Comment by mairaw** [comment]
@rpetrusha thanks for your thorough review. I wasn't much looking at the content inside the sections.

Since we were improving the topics, I also added the VB version of those. Do you wanna double check?

File: docs/framework/network-programming/how-to-download-files-with-ftp.md [hunk 6]
```diff
@@ -1,55 +1,77 @@
...
+
+            reader.Close();
+            response.Close();
+        }
+    }
+}
+```
+
+```vb
+Imports System
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-download-files-with-ftp.md]: Imports System isn't necessary; it's automatically imported by the compiler.

File: docs/framework/network-programming/how-to-download-files-with-ftp.md [hunk 7]
```diff
@@ -1,55 +1,77 @@
...
+Imports System.IO
+Imports System.Net
+
+Namespace Examples.System.Net
+    Public Class WebRequestGetExample
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-download-files-with-ftp.md]: It's typical to do this as a module (`Public Module WebRequestGetExample`)

File: docs/framework/network-programming/how-to-download-files-with-ftp.md [hunk 8]
```diff
@@ -1,55 +1,77 @@
...
+        Public Shared Sub Main()
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-download-files-with-ftp.md]: If this becomes a module, the signature becomes `Public Sub Main()`

File: docs/framework/network-programming/how-to-download-files-with-ftp.md [hunk 9]
```diff
@@ -1,55 +1,77 @@
...
+            Dim responseStream As Stream = response.GetResponseStream()
+            Dim reader As StreamReader = New StreamReader(responseStream)
+            Console.WriteLine(reader.ReadToEnd())
+
+            Console.WriteLine("Download Complete, status {0}", response.StatusDescription)
+
+            reader.Close()
+            response.Close()
+        End Sub
+    End Class
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-download-files-with-ftp.md]: This becomes `End Module`

File: docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md [hunk 10]
```diff
@@ -1,55 +1,77 @@
...
+            Console.WriteLine("Directory List Complete, status {0}", response.StatusDescription);
+
+            reader.Close();
+            response.Close();
+        }
+    }
+}
+```
+
+```vb
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md]: Same comments as previously: System does not need to be imported, and a Module is better Irequiring a change to the Main method signature as well).

File: docs/framework/network-programming/how-to-upload-files-with-ftp.md [hunk 11]
```diff
@@ -48,16 +50,44 @@ namespace Examples.System.Net
...
-## Robust Programming  
-  
-## .NET Framework Security
+        }
+    }
+}
+```
+
+```vb
+Imports System
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-upload-files-with-ftp.md]: Same comments here as for previous examples.

File: docs/framework/network-programming/how-to-download-files-with-ftp.md [hunk 12]
```diff
@@ -1,55 +1,77 @@
...
+            // This example assumes the FTP site uses anonymous logon.
+            request.Credentials = new NetworkCredential("anonymous","janeDoe@contoso.com");
+
+            FtpWebResponse response = (FtpWebResponse)request.GetResponse();
+
+            Stream responseStream = response.GetResponseStream();
+            StreamReader reader = new StreamReader(responseStream);
+            Console.WriteLine(reader.ReadToEnd());
+
+            Console.WriteLine("Download Complete, status {0}", response.StatusDescription);
```
Comment (rpetrusha) [inline docs/framework/network-programming/how-to-download-files-with-ftp.md]: These Console.WriteLine statements with composite formatting strings could be done as interpolated strings in C# and VB if you'd like to make the changes. Here, `Console.WriteLine($"Download Complete, status {response.StatusDescription}");

**Review by rpetrusha — COMMENTED** [review]
Left a number of additional suggestions, @mairaw. Other than removing Imports System, which I'd recommend strongly, you can decide whether you want to do them or not.

File: docs/framework/network-programming/how-to-download-files-with-ftp.md (hunk 12 above)
Comment (mairaw) [reply docs/framework/network-programming/how-to-download-files-with-ftp.md]: are we adding notes about C# version needed when adding interpolated strings?

File: docs/framework/network-programming/how-to-download-files-with-ftp.md (hunk 7 above)
Comment (mairaw) [reply docs/framework/network-programming/how-to-download-files-with-ftp.md]: changing to the typical VB template Module WebRequestGetExample and Sub Main()

**Comment by mairaw** [comment]
Appreciate your comments @rpetrusha. I believe all the feedback is now addressed.

File: docs/framework/network-programming/how-to-download-files-with-ftp.md (hunk 12 above)
Comment (rpetrusha) [reply docs/framework/network-programming/how-to-download-files-with-ftp.md]: Sorry, @mairaw. I missed your question. No, we haven't been, other than in the C# and VB interpolated string docs themselves.

**Pull request merged**